trikot-viewmodels-declarative-flow/compose-flow/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/compose/viewmodel/VMDImage.kt

```diff
@@ -210,6 +223,93 @@ fun VMDImage(
     }
 }
 
+@Composable
+fun VMDImage(
+    modifier: Modifier = Modifier,
+    viewModel: VMDImageViewModel,
+    alignment: Alignment = Alignment.Center,
+    contentScale: ContentScale = ContentScale.Fit,
+    placeholderContentScale: ContentScale = contentScale,
+    alpha: Float = DefaultAlpha,
+    colorFilter: ColorFilter? = null,
+    error: @Composable (placeholderImageResource: VMDImageResource) -> Unit = { imageResource ->
```

Comment:
could we merge error and loading and offer "placeholder" the state (loading or error) could be a parameter to the block?